src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -12,21 +12,37 @@ import {useMeetingControl} from '../hooks';
  * @param {object} props  Data passed to the component
  * @param {string} props.type  Name of the control as defined in adapter
  * @param {string} props.meetingID  ID of the meeting
+ * @param {boolean} props.compact  Flag indicating whether to compact the label
  * @returns {object} JSX of the component
  */
-export default function WebexMeetingControl({type, meetingID}) {
-  const [action, display] = useMeetingControl(type, meetingID);
-  const {icon, text, tooltip} = display;
+
+function renderDropdown(action, display) {
+  const {options, selected} = display;
+
+  return (
+    <select id="dropdown-control" onChange={(event) => action(event.target.value)} value={selected}>
+      {
+        options.map((option) => (
+          <option key={option.deviceId} value={option.deviceId}>
```

Comment:
a dropdown control is generic, we can't expect a `deviceId` property. It should be `value`.